plugins/emoticons.md

```diff
@@ -107,6 +133,25 @@ tinymce.init({
 });
 ```
 
+### `emoticons_images_url`
+
+This option provides the base URL for the images used to represent emojis when using the `emojiimages` database.
```

Comment:
Do we want to document where/how to get the twemoji graphics to self-host them?

Reply:
yes please.

Reply:
Also, maybe:
```suggestion
This option sets the base URL for the images used to represent emojis when using the `emojiimages` database.
```